### ISSUE TITLE
🐛 BUG: [C3] What's the difference between `--● Website or web app` and `--○ Common Worker functions`? 

### ISSUE THREAD

**Issue by rozenmd** [cloudflare/workers-sdk#3326]
### Which Cloudflare product(s) does this pertain to?

C3, Wrangler

### What version of `Wrangler` are you using?

3.0.1

### What operating system are you using?

Mac

### Describe the Bug

With `wrangler init`, I used to be able to create a Worker that could either function as a website or as a worker function - why do I have to choose with C3?

**Comment by penalosa** [comment]
"Hello World" will give you a basic worker, and Common Worker functions will give you a single worker split across many files. Website or web app will generally give you a Pages project

**Comment by admah** [comment]
@rozenmd what would your recommendation be? Keep in mind our goal is a product that allows users to create and deploy both Workers and Pages projects without focusing on the terms "Workers" or "Pages".

**Comment by rozenmd** [comment]
> without focusing on the terms "Workers" or "Pages"
Why?

I get that we're converging the two things to use the same tooling etc, but there are fundamentally different use cases here that don't converge (in my mental model) - one is Functions that run on the Edge (which itself branches off into dozens of use-cases like A/B testing, HTML rewriting, API, etc), and the other is web apps

If you offer someone that wants to deploy a web app a tiny function, they'll be confused just as if you offered someone who wants to deploy a tiny function the scaffolding to build a web app

**Comment by admah** [comment]
@rozenmd what options would you provide to users then? I don't think you answered my original question.

**Comment by rozenmd** [comment]
Good point - I posted this issue from the perspective of a user, I didn't have solutions in mind.

As an existing wrangler init user, there's a bit of confusion going from a wizard that walks you through putting together your project, to one that assumes you already know what each template is
